src/odemis/gui/util/img.py

```diff
@@ -1208,6 +1210,8 @@ def spectrum_to_export_data(proj, raw, vp=None):
             # Limit to the displayed ranges
             range_x = vp.hrange.value
             range_y = vp.vrange.value
+            if min(range_y) < 10:
+                range_y = (0, max(range_y))
```

Comment:
What is this for? range is either in m or pixels. Why 10 m? Why 10 px? 
Typically, when the range is in px, half of the range is negative (eg, -256 -> 256). So cutting half of it doesn't make sense.
This line is really fishy, please explain.